WebDriverAgentLib/Categories/XCUIElement+FBWebDriverAttributes.m

```diff
@@ -28,24 +28,27 @@ @implementation XCUIElement (WebDriverAttributesForwarding)
 
 - (id<FBXCElementSnapshot>)fb_snapshotForAttributeName:(NSString *)name
 {
+  BOOL inDepth = [name isEqualToString:FBStringify(XCUIElement, isWDAccessible)]
+    || [name isEqualToString:FBStringify(XCUIElement, isWDAccessibilityContainer)]
+    || [name isEqualToString:FBStringify(XCUIElement, wdIndex)];
```

Comment:
Just a question (for intention): To simplify setting `inDepth` for all cases?

I mean, instead of setting `isDepth` below (for isWDVisible and isWDAccessible|isWDAccessibilityContainer) with explicit YES/NO, giving `inDepth` for all to simplify the look.

Reply:
makes sense. Simplified.